### DISCUSSION TITLE
upx下载失败，手动下载也各种对不上，好像经常遇到这问题，是upx项目的问题吗？

### DISCUSSION THREAD

**Discussion by sss132919** [coolsnowwolf/lede#11814] [Bugs / 问题反馈]
upx下载失败，手动下载各种对不上，好像经常遇到这问题，是upx项目的问题吗？

**Comment by lirg-111** [comment]
upx

修改makefile
PKG_VERSION:=4.2.2
PKG_HASH:=42ee0455eea610ef7ee732aa1f657b34a351ebcfa64a24c1e2a7aaec74c1e038
